.github/workflows/lint_random_files.yml

```diff
@@ -335,31 +335,31 @@ jobs:
           done
```

Comment:
I suggest moving this logic to a separate Bash script. This will make refactoring and linting easier.